grpc/wrappers.go

```diff
@@ -101,14 +101,29 @@ func NewPublisherClientWrapper(inner pubPB.PublisherClient, timeout time.Duratio
 	return &PublisherClientWrapper{inner, timeout}
 }
 
-// SubmitToCT makes a call to the gRPC version of the publisher
+// SubmitToCT makes a call to the gRPC version of the publisher to send the
+// provided certificate to all of the configured CT logs
 func (pc *PublisherClientWrapper) SubmitToCT(ctx context.Context, der []byte) error {
 	localCtx, cancel := context.WithTimeout(ctx, pc.timeout)
 	defer cancel()
 	_, err := pc.inner.SubmitToCT(localCtx, &pubPB.Request{Der: der})
 	return err
 }
 
+// SubmitToSingleCT makes a call to the gRPC version of the publisher to send
+// the provided certificate to the log specified by log URI and public key
+func (pc *PublisherClientWrapper) SubmitToSingleCT(ctx context.Context, logURL, logPublicKey string, der []byte) error {
+	localCtx, cancel := context.WithTimeout(ctx, pc.timeout)
+	defer cancel()
+	_, err := pc.inner.SubmitToSingleCT(
+		localCtx,
+		&pubPB.Request{
+			LogURL:       &logURL,
+			LogPublicKey: &logPublicKey,
+			Der:          der})
+	return err
```

Comment:
@jsha Aha! Thanks for the pointer about merging in master to get those funcs in-package.

@rolandshoemaker Resolved this feedback in https://github.com/letsencrypt/boulder/pull/2342/commits/ed40b60af48ae4ff855fa500f764d073a949a832